doc/man/julia.1

```diff
@@ -130,7 +130,7 @@ Enable or disable color text
 Load or save history
 
 .TP
---compile={yes|no|all}
+--compile={yes|no|all|min}
 Enable or disable compiler, or request exhaustive compilation
```

Comment:
The description probably needs an update as well.

Reply:
Let me make a new commit and pull request.

Reply:
You can just push more commits here and it can be squashed upon merging.

Reply:
It looks like you did this with the GitHub web interface — you can just keep editing here:

https://github.com/derekmahar/julia/edit/patch-1/doc/man/julia.1?pr=%2FJuliaLang%2Fjulia%2Fpull%2F36585

(Welcome, by the way!)